packages/widget/src/components/common/buttons/button.ts

```diff
@@ -0,0 +1,55 @@
+import type { HTMLTemplateResult } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { html } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { when } from 'lit/directives/when.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { BaseComponent } from '../base-component/base-component';
+import { loaderIcon } from '../../../assets';
+import { buttonStyles } from './button.styles';
+
+@customElement('sygma-button')
```

Comment:
perhaps we should use a more semantic name, since this is not a plain button, but rather one that reflect to loading state

Reply:
sygma-action-button?